### PR TITLE
Port publish code coverage to Dev14 Update 3

### PR DESCRIPTION
**Pull request by nigurr**
* Port Publish Code Coverage Task for Dev14 U3

### CODE REVIEW

File: Tasks/PublishCodeCoverageResults/task.json [hunk 1]
```diff
@@ -27,8 +26,8 @@
             "defaultValue": "JaCoCo",
             "helpMarkDown": "Select the tool with which code coverage summary files have been generated",
             "options": {
-                "JaCoCo": "JaCoCo",
-                "Cobertura": "Cobertura"
+                "Cobertura": "Cobertura",
```
Comment (acesiddhu) [inline Tasks/PublishCodeCoverageResults/task.json]: why this change

File: package.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "through2": "^0.6.3",
     "typescript": "1.8.0",
     "validator": "^3.33.0",
-    "vsts-task-lib": "^0.5.6"
+    "vsts-task-lib": "^0.6.2"
```
Comment (acesiddhu) [inline package.json]: are we sure this wont break compatibility for other tasks?

File: Tasks/PublishCodeCoverageResults/task.json (hunk 1 above)
Comment (nigurr) [inline Tasks/PublishCodeCoverageResults/task.json]: copied from master  🔘

**Comment by AshwiniChalla** [comment]
approved

File: package.json (hunk 2 above)
Comment (nigurr) [inline package.json]: vsts-task-lib don't remove functionality or change. 99.99% it shouldn't. Without this version, this task won't run.

File: Tasks/PublishCodeCoverageResults/task.json (hunk 1 above)
Comment (AshwiniChalla) [inline Tasks/PublishCodeCoverageResults/task.json]: alphabetical order

File: package.json (hunk 2 above)
Comment (AshwiniChalla) [inline package.json]: yes. lets be sure of this

Comment (nigurr) [inline package.json]: I have gone through all changes in vsts-task-lib. it's LIB functionality, nothing related to tasks changed
